src/components/MainPage/Main.js

```diff
@@ -1,59 +1,68 @@
 import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
-import categories from '../../datasets/categories';
+import PropTypes from 'prop-types';
+import categories from '../../utilities/categories';
 import utils from '../../utilities/utils';
 
-function Main({ location }) {
+function Main (
```

Comment:
Really appreciate the way you've formatted things to be as readable as possible.